checkov/common/sca/output.py

```diff
@@ -160,8 +166,12 @@ def create_report_cve_record(
         "description": description,
         "risk_factors": vulnerability_details.get("riskFactors"),
         "published_date": vulnerability_details.get("publishedDate")
-        or (datetime.now() - timedelta(days=vulnerability_details.get("publishedDays", 0))).isoformat(),
+                          or (datetime.now() - timedelta(
+            days=vulnerability_details.get("publishedDays", 0))).isoformat(),
         "licenses": licenses,
+        "root_package_alias": root_package_alias,
```

Comment:
adding `root_package_alias` to every record will add an extra space that maybe in the future we will need it for other things, when we can create it when we need it using the package name and version.